demo/mobile/config.xml

```diff
@@ -1,93 +1,71 @@
+    <preference name="webviewbounce" value="false" />
+    <preference name="UIWebViewBounce" value="false" />
+    <preference name="DisallowOverscroll" value="true" />
+    <preference name="android-minSdkVersion" value="16" />
+    <preference name="BackupWebStorage" value="none" />
+    <feature name="StatusBar">
+        <param name="ios-package" onload="true" value="CDVStatusBar" />
+    </feature>
+    <platform name="android">
+        <icon density="ldpi" src="www/res/icons/android/icon.png" />
+        <icon density="mdpi" src="www/res/icons/android/icon-mdpi.png" />
+        <icon density="hdpi" src="www/res/icons/android/icon-hdpi.png" />
+        <icon density="xhdpi" src="www/res/icons/android/icon-xhdpi.png" />
+        <icon density="xxhdpi" src="www/res/icons/android/icon-xxhdpi.png" />
+        <icon density="xxxhdpi" src="www/res/icons/android/icon-xxxhdpi.png" />
+        <splash density="land-hdpi" src="www/res/splash/android/splash-land-hdpi.png" />
+        <splash density="land-ldpi" src="www/res/splash/android/splash-land-ldpi.png" />
+        <splash density="land-mdpi" src="www/res/splash/android/splash-land-mdpi.png" />
+        <splash density="land-xhdpi" src="www/res/splash/android/splash-land-xhdpi.png" />
+        <splash density="port-hdpi" src="www/res/splash/android/splash-port-hdpi.png" />
+        <splash density="port-ldpi" src="www/res/splash/android/splash-port-ldpi.png" />
+        <splash density="port-mdpi" src="www/res/splash/android/splash-port-mdpi.png" />
+        <splash density="port-xhdpi" src="www/res/splash/android/splash-port-xhdpi.png" />
+        <preference name="SplashScreen" value="screen" />
+    </platform>
+    <platform name="ios">
+        <icon height="180" src="www/res/icons/ios/Icon-60@3x.png" width="180" />
+        <icon height="60" src="www/res/icons/ios/Icon-60.png" width="60" />
+        <icon height="120" src="www/res/icons/ios/Icon-60@2x.png" width="120" />
+        <icon height="76" src="www/res/icons/ios/Icon-76.png" width="76" />
+        <icon height="152" src="www/res/icons/ios/Icon-76@2x.png" width="152" />
+        <icon height="40" src="www/res/icons/ios/Icon-40.png" width="40" />
+        <icon height="80" src="www/res/icons/ios/Icon-40@2x.png" width="80" />
+        <icon height="57" src="www/res/icons/ios/icon.png" width="57" />
+        <icon height="114" src="www/res/icons/ios/icon@2x.png" width="114" />
+        <icon height="72" src="www/res/icons/ios/Icon-72.png" width="72" />
+        <icon height="144" src="www/res/icons/ios/Icon-72@2x.png" width="144" />
+        <icon height="29" src="www/res/icons/ios/Icon-small.png" width="29" />
+        <icon height="58" src="www/res/icons/ios/Icon-small@2x.png" width="58" />
+        <icon height="50" src="www/res/icons/ios/Icon-50.png" width="50" />
+        <icon height="100" src="www/res/icons/ios/Icon-50@2x.png" width="100" />
+    </platform>
+    <engine name="android" spec="^6.2.3" />
```

Comment:
It's better not to have spec on the engine tags so it uses the one pinned on the cordova-lib version used to build the app. 
This way we avoid incompatibilities between an old platform and a new cordova-lib, or a new platform and an old cordova-lib.